key_value_store.cc

```diff
@@ -0,0 +1,34 @@
+#include "key_value_store.h"
+
+#include <optional>
+#include <functional>
+#include <string>
+#include <vector>
+#include <iostream>
+
+KeyValueStore::KeyValueStore() : key_value_store_() {}
+
+KeyValueStore::~KeyValueStore() {}
+
+std::optional<std::string> KeyValueStore::Get(const std::string &key) {
+  auto it = key_value_store_.find(key);
+
+  if (it != key_value_store_.end()) {
+    if (!it->second.empty()) {
```

Comment:
Does this violate the API slightly?  I don't remember there being a restriction on inserting an empty string value, only that an empty key isn't allowed.

Reply:
you are right. I will fix